deploy/operator.yaml

```diff
@@ -16,7 +16,7 @@ spec:
       containers:
         - name: ibm-mongodb-operator
           # Replace this with the built image name
-          image: quay.io/blackhole/ibm-mongodb-operator:latest
+          image: quay.io/opencloudio/ibm-mongodb-operator:latest
```

Comment:
can you leave this unchanged?

Reply:
@chenzhiwei `blackhole` is my test repo, why need to keep it.